src/redux/categories/categories.js

```diff
@@ -0,0 +1,52 @@
+const ADD = 'my-app/categories/ADD';
+const REMOVE = 'my-app/categories/REMOVE';
```

Comment:
- [ ] Acoording to project requirement, `categories.js` should have only an action`CHECK_STATUS` kindly remove this as it is not needed